src/main/java/core/basesyntax/BuilderTestApp.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
 public class BuilderTestApp {
-  public static void main(String[] args) {
-      // Remove this comment and test your Builder implementation here...
-  }
+
+    public static void main(String[] args) {
+
```

Comment:
the empty line under the method signature is redundant

Reply:
fixed!